src/index.js

```diff
@@ -1,18 +1,31 @@
 import 'assets/styles/main.scss';
-import cloudinary from 'cloudinary-core';
 import VideoPlayer from './video-player';
 import { assign } from 'utils/assign';
 
-cloudinary.VideoPlayer = VideoPlayer;
+if (window.cloudinary && window.cloudinary.Cloudinary) {
+  console.error('For version 2.0.0 and above, cloudinary-core is not needed for using the Cloudinary Video Player');
+}
 
-const proto = cloudinary.Cloudinary.prototype;
+const getConfig = (playerOptions = {}, cloudinaryConfig) => assign(playerOptions, { cloudinaryConfig: cloudinaryConfig || playerOptions });
 
-proto.videoPlayer = function(id, options = {}, ready = null) {
-  assign(options, { cloudinaryConfig: this });
-  return new VideoPlayer(id, options, ready);
-};
+const getVideoPlayer = (config) => (id, playerOptions, ready) => new VideoPlayer(id, getConfig(playerOptions, config), ready);
+
+const getVideoPlayers = (config) => (selector, playerOptions, ready) => VideoPlayer.all(selector, getConfig(playerOptions, config), ready);
+
+const cloudinaryVideoPlayerConfig = (config) => ({
+  videoPlayer: getVideoPlayer(config),
+  videoPlayers: getVideoPlayers(config)
+});
 
-proto.videoPlayers = function(selector, options = {}, ready = null) {
-  assign(options, { cloudinaryConfig: this });
-  return VideoPlayer.all(selector, options, ready);
+export const videoPlayer = getVideoPlayer();
+export const videoPlayers = getVideoPlayers();
+
+window.cloudinary = {
+  ...(window.cloudinary || {}),
```

Comment:
Make sure the spread operator is transpiled for ie11 support

Reply:
The code is getting compiled.